stages/eib_image

```diff
@@ -60,9 +60,14 @@ if [[ "${EIB_ARCH}" == "armhf" ]] || [[ "${EIB_ARCH}" == "arm64" ]]; then
   # Empty uEnv.txt otherwise ostree gets upset
   > "${BOOT}"/loader/uEnv.txt
   ln -s loader/uEnv.txt "${BOOT}"/uEnv.txt
+  ostree --repo="${REPOPATH}" config set sysroot.bootloader uboot
 else
   # Assume grub for all other architectures
```

Comment:
This is not part of the changes here, but this comment is not true anymore since PAYG images use sd-boot.